src/main/java/loanbook/commons/core/Messages.java

```diff
@@ -12,4 +12,16 @@
     public static final String MESSAGE_INVALID_PASSWORD = "The password provided is incorrect!";
     public static final String MESSAGE_INVALID_OLD_PASS = "The old password entered is incorrect!";
     public static final String MESSAGE_SAME_AS_CURRENT_PASSWORD = "The password provided is same as before!";
+    public static final String MESSAGE_AUTHEN_FAILURE = "Your password might be wrong "
+            + "or you did not enable \"less secure app\" in your google account setting before you use remind command."
+            + " Please refer to the User Guide!";
+    public static final String MESSAGE_NO_NETWORK_CONNECTION = "No connection to the network! "
+            + "Please make sure you have network connected!";
+    public static final String MESSAGE_BAD_RUNTIME = "You execute your code in a Java runtime"
+            + " that does not support UTF-8!";
```

Comment:
How should a user fix this? Can we add this to the developer/user guide to explain? No change needed for code.

Reply:
Ok, will add to the UGDG. Thank you so much!